fixit/cli/main.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Usage:
+#
+#   $ fixit run_rules --help
+#   $ fixit apply_fix --help
+#   $ fixit insert_suppressions --help
+#   $ fixit add_new_rule --help
+
+import argparse
+import importlib.util
+from importlib.machinery import ModuleSpec
+from pathlib import Path
+from types import ModuleType
+from typing import List
+
+from absl import app
+from absl.flags import argparse_flags
+
+from fixit.cli import add_new_rule, apply_fix, insert_suppressions, run_rules
+
+
+def get_fixit_version() -> str:
+    """Get current Fixit's version"""
+    directory = Path(__file__).resolve().parents[1]
+
+    spec: ModuleSpec = importlib.util.spec_from_file_location(
+        "version", directory / "_version.py"
+    )
+
+    version: ModuleType = importlib.util.module_from_spec(spec)
+    # pyre-ignore Pyre doesn't know about importlib entirely.
+    spec.loader.exec_module(version)
+    # pyre-ignore Pyre has no way of knowing that this constant exists.
+    return version.FIXIT_VERSION
```

Comment:
Can we simply do `from fixit._version import FIXIT_VERSION`?

Reply:
sure.